src/utils/testUtils.test.js

```diff
@@ -431,3 +432,18 @@ describe("PropStruct", () => {
 			));
 	});
 });
+
+describe("ignoreConsoleError", () => {
+	spyOnConsole(["error"]);
+	it("ignores console error", () => {
```

Comment:
Well, it's great you did it, however, I will say to you what I said to Tony, don't feel obligated to test "testutils" functions.